dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Document</title><script defer="defer" src="main.bundle.js"></script><link href="main.css" rel="stylesheet"></head><body><div id="root"></div></body></html>
```

Comment:
@AStashevskaya Lets remove `dist` directory from Pull Request, since `dist` already added to `.gitignore`.